app/controllers/form_controller.rb

```diff
@@ -89,6 +95,7 @@ def show_page
           if flash[:errors].present?
             restore_previous_errors(flash[:errors])
             restore_error_field_values(flash[:log_data])
+            @questions_with_errors_count = flash[:questions_with_errors_count]
```

Comment:
Given that we now only recover the errors for page, we need to keep track of how many errors were actually added to the log to know if we need to display "see all related answers" flow